docker/Dockerfile.manylinux2010-x64-custom

```diff
@@ -0,0 +1,10 @@
+FROM dockcross/manylinux2010-x64
+
+ENV DEFAULT_DOCKCROSS_IMAGE manylinux2010-x64-custom
+
+ADD https://www.cpan.org/src/5.0/perl-5.28.1.tar.gz .
```

Comment:
So we still need to add perl manually with 2010? Did you try without it?

Reply:
No, I did not.

Reply:
You're right. It's not needed anymore. So I'll remove our custom stuff.